### PR TITLE
Update flashing-vbios-legion

### PR DESCRIPTION
**Pull request by kirakenchin**

### CODE REVIEW

File: _drafts/flashing-vbios-legion.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+1. Download Nvflash from - https://www.techpowerup.com/download/nvidia-nvflash/
+2. Extract nvflash into ``C:\nvflash\``
+  ![image](https://user-images.githubusercontent.com/100846697/210909374-0ce998a5-11cf-4503-84dc-98662aba2ecb.png)
+3. Download VBIOS for your GPU from attachment below
+4. Open CMD as admin, type in cd ``C:\nvflash\``. You can copy the path from address bar in File Explorer
+5. Type dir to ensure you are in the right directory. You should see your ``VBIOS.ro``m and ``nvflashxx.exe`` files
+6. Run these commands
+
+#### Backup
+```nvflash64.exe -b backup.rom```
```
Comment (nonkerdoob) [inline _drafts/flashing-vbios-legion.md:52]: Some weird markup issue on this line.

**Review by nonkerdoob — COMMENTED** [review]
Need to format the links properly.

File: _drafts/flashing-vbios-legion.md (hunk 1 above)
Comment (kirakenchin) [reply _drafts/flashing-vbios-legion.md]: check now and let me know

**Comment by nonkerdoob** [comment]
It's better to use titles for the GPUs as it also serve as navigation points with in page TOC.

**Review by nonkerdoob — APPROVED** [review]

**Pull request merged**